week08_pomdp/practice_pytorch.ipynb

```diff
@@ -172,10 +165,17 @@
     "    def __init__(self, obs_shape, n_actions, reuse=False):\n",
     "        \"\"\"A simple actor-critic agent\"\"\"\n",
     "        super(self.__class__, self).__init__()\n",
+    "        # NOTICE: Adding `Batch Normalization` to your network helps a lot in\n",
+    "        ## preventing gradient exploration and helps to have stable training\n",
+    "        ## and fast convergence\n",
```

Comment:
This is a strong claim as many students have gone through this assignment before and had no issues like this one. Please provide supporting evidence that this is actually needed.

Reply:
As I mentioned previously in the spot, this helps prevent the gradient from exploding; I saw lots of people do the same thing in their assignment one year ago.

You can find it here: https://github.com/exldna/AIIJC-mirror/blob/823ddee2e7cb70f5fb931472227bdfce40d8d5af/source/model/nn_model.py.

And I am sorry about the typo error; It is not **gradient exploration**; it is **gradient explosion**.

Reply:
I adjusted the typo error!